ros/openapi/openapi.json

```diff
@@ -72,6 +72,9 @@
                     },
                     {
                         "$ref": "#/components/parameters/filterStateParam"
+                    },
+                    {
+                        "$ref": "#/components/parameters/filterOSParam"
```

Comment:
Just an observation -
`Advisor` supports filter called `rhel_version` where it accepts version values example - 7.3, 8.5
whereas Patch > Systems supports filter called `os` where it accepts full version name example - RHEL 7.3,RHEL 8.5
Here, you are using filter name as `os` and accepts version values example - 7.3, 8.5

Secondly, on UI side both services are showing full name on filter chip whereas with PR https://github.com/RedHatInsights/ros-frontend/pull/110 change it shows just version value.


For reference attaching screenshots:

Advisor Systems -
![Screenshot from 2022-04-08 14-45-56](https://user-images.githubusercontent.com/6470528/162406653-bfe5c057-2aa0-4436-9b27-6c3123ba28dc.png)

Patch > Systems -
![Screenshot from 2022-04-08 14-47-07](https://user-images.githubusercontent.com/6470528/162406666-8e97c604-979d-41e0-8048-cbc06e6267b2.png)
-45-56.png…]()